### ISSUE TITLE
Define self-description requirements for "Instance of [...]" datasets

### ISSUE THREAD

**Issue by briesenberg07** [uwlib-cams/uwlswd#43]
Related to https://github.com/uwlib-cams/uwlswd/discussions/42 / Escaping the VoID.  
Starting on definition as DCTAP\*\* -- see dctap/[uwlswd_instancesOf.csv](https://github.com/uwlib-cams/uwlswd/blob/main/dctap/uwlswd_instancesOf.csv).

# Proposed changes and questions
- I don't find any property owl version in the [The OWL 2 Schema vocabulary (OWL 2)](https://www.w3.org/2002/07/owl#)! Am I missing this somewhere? I believe we will include version information; but I can't find this property!
- Propose discontinuing use of void:class; resources described in datasets are modeled as members of classes, dataset titles indicate dataset class focus; is this information needed in the dataset self-description?

**Comment by gerontakos** [comment]
Most certainly it's supposed to be owl:versionInfo (i.e. https://www.w3.org/2002/07/owl#versionInfo). However, that may not have been the best choice. Although not limited by range (R=rdfs:Resource), the property is intended (looking at [the spec](https://www.w3.org/TR/owl-ref/#versionInfo-def)) to describe ontologies or OWL ontology constructs. 

[Later]: after looking around, owl:versionInfo seems like the only choice! There's http://purl.org/pav/version but that ontology seems to be hiding in a dumpster in Boston (much like our datasets are hiding in a garbage can in Seattle). Anyway, I'm all-in on owl:versionInfo.

**Comment by gerontakos** [comment]
As for void:class, that's useful information in the current distribution, and is much better data than the titles with their free text values. I think an important problem to solve revolves around our desire to deprecate the dataset uwlib.55.a#uwSemWeb. So, to begin: what is a void:class? It is the class of all entities described in a dataset that is a subset of another dataset; the parent dataset has class-based partitions. Using void:class was an excellent choice when we wanted to offer our DPLA data as unified beyond the assertions in the Aggregations. However, if we deprecate uwlib.55.a#uwSemWeb, the partitions are no longer partitions and it would be irregular, therefore, to describe the datasets as subsets using void:class. So I'll take a look at discussion 42, as I believe that's the appropriate place to explore this problem. However, here, at issue 43, I will only say that if the partitions remain partitions of 55.a#uwSemWeb, then I think void:class should remain; if, however, we eliminate 55.a#uwSemWeb, then we should no longer use void:class.

**Comment by briesenberg07** [comment]
Thank you @gerontakos and @cspayne 

> Although not limited by range (R=rdfs:Resource), the property is intended (looking at [the spec](https://www.w3.org/TR/owl-ref/#versionInfo-def)) to describe ontologies or OWL ontology constructs.

That's what gave me pause when I looked at owl:versionInfo! I don't think our datasets are OWL constructs... (maybe it doesn't matter?)

Create UWLSWD property `versionNumber` or similar??

**Comment by gerontakos** [comment]
I don't think so. W3C's Data on the Web BPs, section 8.6 (see [example 7](https://www.w3.org/TR/dwbp/#dataVersioning)), sort-of suggests we use either owl:versionInfo or pav:version. If it's good enough for them, it's good enough for me. (hashtag bornToFollow)

**Comment by briesenberg07** [comment]
Picking up from @cspayne 's [proposed starting point](https://github.com/uwlib-cams/uwlswd/discussions/42#discussioncomment-7321042) in defining requirements for metadata in the "Instances of (...)" resources, copied below with comments:

| By DataCite:  | By UWLSWD: |
|---|---|
|- dct:title|- dc:language |
|- dct:creator OR dc:creator|- dct:license |
|- dct:publisher OR dc:publisher|- dct:description |
|- dct:issued|- dct:format | 
|- schema:disambiguatingDescription| - dct:hasFormat (can be added during serialization)|
||- schema:version|

@briesenberg07 comments:
- [dct:language](http://purl.org/dc/terms/language) seems preferable to [dc:language](http://purl.org/dc/elements/1.1/language), updated [instances_of.csv](https://github.com/uwlib-cams/uwlswd/blob/main/dctap/instances_of.csv) uses the term, not the element (unless there was a reason why we prefer the element?)
- Note that discussion above refers to using `owl:versionInfo` but I'm pretty sure we've settled on `schema:version` for use
- DCTAP still needs work - still missing cardinality, may have errors, etc. (and of course we haven't agreed on these triples yet!)

**Comment by cspayne** [comment]
>* [dct:language](http://purl.org/dc/terms/language) seems preferable to [dc:language](http://purl.org/dc/elements/1.1/language), updated [instances_of.csv](https://github.com/uwlib-cams/uwlswd/blob/main/dctap/instances_of.csv) uses the term, not the element (unless there was a reason why we prefer the element?)

I don't think there is a specific reason why we prefer dc:language. DataCite and Schema.org both recommend using a language code from  IETF BCP 47, but it looks like that is acceptable with the dct:language element as well. 
If we do make that change to our mapping, the scripts will need small updates.

**Comment by gerontakos** [comment]
No big deal from my view; we just have to make sure we either use data IRIs or BCP47 literals. If we can manage that, we're good with dct.date.

**Comment by cspayne** [comment]
An update!

- The triples we identified in our DC Tap [instances_of.csv](https://github.com/uwlib-cams/uwlswd/blob/main/dctap/instances_of.csv) have been added to the "instance of ..." datasets. 
- void:inDataset triples were removed
- versions were updated
- metadata has been generated and uploaded to DataCite, and DataCite URLs were updated. 
- resources have been serialized and pushed to GitHub

_No changes were made to the void_description file_, so it is behind on version numbers.
